tests/search/global-phase-ranking/answers/first-phase-as-rank-feature.json

```diff
@@ -0,0 +1,119 @@
+{
+  "root": {
+    "id": "toplevel",
+    "relevance": 1,
+    "fields": {
+      "totalCount": 5
+    },
+    "coverage": {
+      "coverage": 100,
+      "documents": 5,
+      "full": true,
+      "nodes": 1,
+      "results": 1,
+      "resultsFull": 1
+    },
+    "children": [
+      {
+        "id": "id:test:test::5",
+        "relevance": 5,
+        "source": "search",
+        "fields": {
+          "matchfeatures": {
+            "firstPhase": -5
+          },
+          "sddocname": "test",
+          "documentid": "id:test:test::5",
+          "doc_vec": {
```

Comment:
Only some of the fields in this expected file are used by the test. Consider adding a _minimal_ summary class in the schema file and use _&summary=minimal_ in the query to reduce the size of this file down to what is actually used:
```
document-summary minimal {
    summary documentid type string {}
}
```